app/views/layouts/application.html.haml

```diff
@@ -1,7 +1,7 @@
 !!!
 %html
   %head
-    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+    %meta{ :content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```

Comment:
Hash attribute should end with one space before the closing brace<br>Line is too long. [83/80]